draft-specifications/CapTP Specification.md

```diff
@@ -689,6 +689,90 @@ resolver when a resolution is available to the `listen-desc` object. If a
 resolution is already available, the resolver provided in `listen-desc` MUST be
 fulfilled or broken.
 
+## `op:get`
+
+`op:get` requests the value for the named field of an eventually settled
+[Struct](Model.md#Struct).  The get operation follows promise resolutions,
+inheriting the rejection reason of any intermediate rejected promise.
+The operation rejects the answer if the ultimate fulfillment of the receiver
+is not a struct.
+
+The messages looks like:
+```
+<op:get <receiver-pos>       ; <desc:answer | desc:import-promise>
```

Comment:
Editorial note: I chose `receiver-pos` here, deviating from the precedent in `op:pick`, because `promise-pos` was very specific to extracting from multi-value fulfillments. `struct-pos` would be consistent with that convention.

Reply:
nit: receiver-pos sounds like it takes a positive integer, I think receiver-desc would be a better name, WDYT?

Reply:
Second nit, I'll put it here but it applies to all arguments in the operators introduced, I think arguments shouldn't have `<>` around them (i.e. `<op:get receiver-pos field-name new-answer-pos>`)